recipes/libradolan/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+src_dir="$(pwd)"
+mkdir ../build
+cd ../build
+cmake $src_dir -DCMAKE_INSTALL_PREFIX=$PREFIX
+make
+make install
```

Comment:
This is how it should be done according to the README:

- cmake
- make
- make install

cmake takes care of configuring everything, AFAIK, and creates the Makefile

I'm currently testing locally, because there is some problem with the CMakeLists.txt